source/appendices/a-glossary-of-terms.rst

```diff
@@ -1125,15 +1125,15 @@ Note: The main Code of Practice document takes precedence in the case of any con
      -
 
    * - TR_B1
-     - Book Requests (Controlled). A pre-set Standard View of TR showing full text activity for all book content which is not Open or Free_To_Read.
+     - Book Requests (Controlled). A pre-set Standard View of TR showing full text activity for all book and reference work content which is not Open or Free_To_Read.
      -
 
    * - TR_B2
-     - Book Access Denied. A pre-set Standard View of TR showing where users were denied access because simultaneous-use (concurrency) licenses were exceeded, or their institution did not have a license for the book.
+     - Book Access Denied. A pre-set Standard View of TR showing where users were denied access because simultaneous-use (concurrency) licenses were exceeded, or their institution did not have a license for the book or reference work.
      -
 
    * - TR_B3
-     - Book Usage by Access Type. A pre-set Standard View of TR showing all applicable Metric_Types broken down by Access_Type.
+     - Book Usage by Access Type. A pre-set Standard View of TR showing all applicable Metric_Types broken down by Access_Type for books and reference works.
```

Comment:
I'm wondering why you're using "books and reference works" in the glossary, but "book (Data_Types Book and Reference_Work)" in Section 4.3? I prefer "books and reference works".